evaluation/scripts/simple_benchmark.py

```diff
@@ -31,7 +61,55 @@ def main():
     )
     logger.setLevel(logging.INFO)
 
-    logger.info('Beginning evaluation')
+    # set random seed
+    set_seed(eval_args.random_seed)
+
+    logger.info("Beginning evaluation")
+
+    # Load model & tokenizer
+    logger.info("Loading model...")
+    tokenizer = AutoTokenizer.from_pretrained(eval_args.tokenizer_name or eval_args.model_name_or_path)
+    tokenizer.pad_token = tokenizer.eos_token
+    tokenizer.padding_side = "left"
+
+    model = AutoModelForCausalLM.from_pretrained(eval_args.model_name_or_path, pad_token_id=tokenizer.eos_token)
+    model.config.pad_token_id = model.config.eos_token_id
+    model.resize_token_embeddings(len(tokenizer))
+    model.to(torch_device)
+
+    # Load dataset
+    logger.info("Benchmarking TyDiQA...")
+    target_langs = ["english"]
+    data = load_dataset("tydiqa", "secondary_task", split="validation")
+    dataset = TyDiQADataset(data, tokenizer, target_langs)
+
+    tydiqa_substring_matches = 0
+    for sample in tqdm(dataset):
+        output = model.generate(
+            input_ids=sample["input_ids"].to(torch_device),
+            attention_mask=sample["attention_mask"].to(torch_device),
+            max_length=min(sample["input_len"]*2, model.config.n_positions),
+        )
+
+        prompt_len = len(sample["prompt"])
+        decoded_output = tokenizer.decode(output[0], skip_special_tokens=True)
+        predicted_answer = decoded_output[prompt_len:]
+        
+        target_answers = sample["target_answer"]
+        substring_match = any([target_answer in predicted_answer.lower() for target_answer in target_answers])
+        tydiqa_substring_matches += substring_match
+    tydiqa_metrics = dict(substring_matches=tydiqa_substring_matches / len(dataset) * 100)
```

Comment:
IIRC declaring dictionaries as curly braces are marginally faster than calling `dict()` in Python. I also think they're slightly more readable, though this might be a matter of personal preference. Your call!

```suggestion
    tydiqa_metrics = {"substring_matches": tydiqa_substring_matches / len(dataset) * 100}
```

Reply:
Good suggestion! I think breaking into multiple lines could be even better, just in case that we will have several different metrics for one task and we keep the code format consistent.

```suggestion
    tydiqa_metrics = {
        "substring_matches": tydiqa_substring_matches / len(dataset) * 100
    }
```